packages/gestalt/src/TextField.js

```diff
@@ -40,6 +42,7 @@ type Props = {|
     value: string,
   |}) => void,
   placeholder?: string,
+  tags?: Array<Node>,
```

Comment:
could this be `Array<Element<typeof Tag>>`

Reply:
oh nice, I couldn't figure out how to do this